.github/workflows/windows.yml

```diff
@@ -90,13 +117,24 @@ jobs:
       # For some reason WiX isn't currently on the GitHub runner path. This is a
       # temporary workaround until that is fixed.
       - name: Add WiX to Path
+        if: ${{ ! inputs.self-hosted-runner }}
         run: |
           "$env:WIX\bin" >> $env:GITHUB_PATH
+
+      # Configure tools in a self-hosted runner.
+      - name: Add python and wixtoolset to path
+        if: ${{ inputs.self-hosted-runner }}
+        run: |
+          "$env:LOCALAPPDATA\Programs\Python\Python312" >> $env:GITHUB_PATH
```

Comment:
This version is different than on github runners: https://github.com/servo/servo/pull/33081/files#diff-5ce5c9ff86f58b1f87b35b5227b2e84cb69f022d6741e1854f3e1e181091773cR109

Reply:
i’ll bake a new image and fix this :)

Reply:
done in 9057b2074387c18486aab521924c9a4863218f4f ([ci-runners commit](https://github.com/servo/ci-runners/commit/11f14fd0c654cda5ead5997fdfc4b35ee59bed67))